locale/templates/settings.ftl

```diff
@@ -24,6 +24,7 @@
 # This is used to refer to a user's account, e.g. "update your Firefox account ..."
 -product-firefox-account = Firefox account
 product-mozilla-vpn = Mozilla VPN
+product-pocket = Pocket
```

Comment:
Add the PayPal entry here.

Reply:
`-brand-name-paypal` already exists in `main.ftl`.  `main.ftl` and `settings.ftl` serve two different frontends.  (`main.ftl` is a  confusing name. ☹️ )